CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [5.0.1-beta.1](https://github.com/purecloudlabs/iframe-coordinator/compare/v5.0.2-beta.3...v5.0.1-beta.1) (2022-08-30)
```

Comment:
This hasn't been published yet, so I think this is just a result of a local run of the version script. We should probably revert this, as we only want the changelog to update during the CI build where we publish the library.